curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/66c395a16c3ff33dc6f8e7c0.md

```diff
@@ -0,0 +1,132 @@
+---
+id: 66c395a16c3ff33dc6f8e7c0
+title: Step 26
+challengeType: 0
+dashedName: step-26
+---
+
+# --description--
+
+Change `.sort` to `.toSorted`.
+
+The `toSorted` returns a new array with the elements sorted in ascending order. On the other hand, `sort()` method sorts the elements of array in place.
```

Comment:
```suggestion
The `.sort()` method mutates the original array - in other words, it modifies the order of the elements directly. This is generally considered bad practise, as it can result in unexpected side effects.

Instead, you should use the `.toSorted()` method, which creates a new array. Change your `.sort()` call to `.toSorted()`. Do not modify the callback function.
```